File: meeting_3/fooBar.js
function fooBar(num) {
    console.log(`Foo Bar Game of ${num}`);
    for (let i = 1; i <= num; i++) {
        let output = '';
        if (i % 3 == 0) output += 'Foo';
        if (i % 5 == 0) output += 'Bar';
        if (i % 3 != 0 && i % 5 != 0) output += i.toString();
        console.log(output);
    }
}

fooBar(15);
fooBar(42);
fooBar(100);

File: meeting_3/fooBar.cpp
#include <iostream>

using namespace std;

void fooBar(int num);

int main() {    
    fooBar(15);
    fooBar(42);
    fooBar(100);
}

void fooBar(int num) {
    cout << "Foo Bar Game of " << num << endl;
    for (int i = 1; i <= num; i++) {
        if (i % 3 == 0) cout << "Foo";
        if (i % 5 == 0) cout << "Bar";
        if (i % 3 != 0 && i % 5 != 0) cout << i;
        cout << endl;
    }
}